test/bacnet/datalink/bvlc/src/main.c

```diff
@@ -859,14 +859,24 @@ static void test_BVLC_BBMD_Address(void)
     uint8_t apdu[480] = { 0 };
     int apdu_len = 0;
     int test_apdu_len = 0;
-    BACNET_IP_ADDRESS bbmd_address;
+    BACNET_HOST_N_PORT bbmd_address;
     BACNET_IP_ADDRESS test_bbmd_address;
     BACNET_ERROR_CODE error_code = ERROR_CODE_SUCCESS;
+#if 0
     bool status = false;
```

Comment:
I didn't want to touch the bvlc_* api functions as they might be of use to someone - just not really of use here in this test.

Reply:
We probably want to include them to have better Line of Code Coverage (see lcov in Cmake test builds).

Reply:
restored = done.